### PR TITLE
Add description on `admin pause ddl jobs` and `admin resume ddl jobs`

### PR DESCRIPTION
**Pull request by dhysum**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add description about the new admin command:
- admin pause ddl jobs
- admin resume ddl jobs

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
    - https://github.com/pingcap/tidb/pull/43081
    - https://github.com/pingcap/tidb/pull/43159
- Other reference link(s):
    - https://github.com/pingcap/tidb/issues/18015 
    - https://github.com/pingcap/tidb/issues/40041

Related issue number: 
- https://github.com/pingcap/tidb/issues/39751 
- https://github.com/pingcap/tidb/pull/43061

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ddl-introduction.md [hunk 1]
```diff
@@ -178,6 +178,14 @@ absent -> delete only -> write only -> write reorg -> public
 
     取消一个已经执行完成的 DDL 任务会在 `RESULT` 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
 
+- `ADMIN PAUSE DDL JOBS job_id [, job_id]`：用于暂停正在执行的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务暂停执行。（实验特性）
```
Comment (ran-huang) [inline ddl-introduction.md:181]:
```suggestion
- `ADMIN PAUSE DDL JOBS job_id [, job_id]`：用于暂停正在执行的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务暂停执行。详情参阅 [`ADMIN PAUSE DDL JOBS`](/sql-statements/sql-statement-admin-pause-ddl.md)。（实验特性）
```

File: ddl-introduction.md [hunk 2]
```diff
@@ -178,6 +178,14 @@ absent -> delete only -> write only -> write reorg -> public
...
+
+    只有处于执行中或仍在等待中的 DDL 任务可以暂停，否则会在 `RESULT` 列看到 `Job 3 can't be paused now`。
+
+- `ADMIN RESUME DDL JOBS job_id [, job_id]`：用于恢复已被暂停的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务正常执行。（实验特性）
```
Comment (ran-huang) [inline ddl-introduction.md:185]:
```suggestion
- `ADMIN RESUME DDL JOBS job_id [, job_id]`：用于恢复已被暂停的 DDL 任务。执行该命令后，执行 DDL 任务的 SQL 语句体现为正在执行，后台任务正常执行。详情参阅 [`ADMIN RESUME DDL JOBS`](/sql-statements/sql-statement-admin-resume-ddl.md)。（实验特性）
```

File: TOC.md [hunk 3]
```diff
@@ -684,6 +684,8 @@
       - [`ADD INDEX`](/sql-statements/sql-statement-add-index.md)
       - [`ADMIN`](/sql-statements/sql-statement-admin.md)
       - [`ADMIN CANCEL DDL`](/sql-statements/sql-statement-admin-cancel-ddl.md)
+      - [`ADMIN PAUSE DDL`](/sql-statements/sql-statement-admin-pause-ddl.md)
+      - [`ADMIN RESUME DDL`](/sql-statements/sql-statement-admin-resume-ddl.md)
```
Comment (ran-huang) [inline TOC.md:688]: 这里麻烦按字母顺序排列哈

File: sql-statements/sql-statement-admin-pause-ddl.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+---
+title: ADMIN PAUSE DDL
+summary: TiDB 数据库中 ADMIN PAUSE DDL 的使用概况。
+---
+
+# ADMIN PAUSE DDL
+
+`ADMIN PAUSE DDL` 语句用于暂停当前正在运行的 DDL 作业。可以通过 [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) 语句获取 DDL 作业的 `job_id`。
+
+用于暂停已经提交但未执行完成的 DDL 任务。成功暂停后，执行 DDL 任务的 SQL 语句不会立即返回，表现为正在执行。暂停一个已经执行完成的 DDL 任务会在 RESULT 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-pause-ddl.md:10]:
```suggestion
该语句可用于暂停已经提交但未执行完成的 DDL 任务。成功暂停后，执行 DDL 任务的 SQL 语句不会立即返回，表现为正在执行。如果尝试暂停一个已经完成的 DDL 任务，会在 `RESULT` 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
```

File: sql-statements/sql-statement-admin-pause-ddl.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+NumList ::=
+    Int64Num ( ',' Int64Num )*
+```
+
+## 示例
+
+可以通过 `ADMIN PAUSE DDL JOBS` 语句暂停当前正在运行的 DDL 作业，并返回对应作业是否暂停成功，直到被 `ADMIN RESUME DDL JOBS` 恢复：
+
+{{< copyable "sql" >}}
+
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-pause-ddl.md:27]:
```suggestion
```

File: sql-statements/sql-statement-admin-pause-ddl.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+如果暂停失败，会显示失败的具体原因。
+
+> **注意：**
+>
+> + 该操作可以暂停 DDL 作业，除版本升级外，其他操作和环境变更（例如机器重启、集群重启）不会暂停 DDL 作业。
+> + 版本升级操作发生时，会将正在运行的 DDL 作业暂停；并将升级过程中发起的 DDL 作业也暂停。升级结束后，将所有暂停的 DDL 作业恢复运行。
+> + 该操作可以同时暂停多个 DDL 作业，可以通过 [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) 语句来获取 DDL 作业的 `job_id`。
+> + 如果希望暂停的作业已经执行完毕或接近执行完毕，暂停操作将失败。
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-pause-ddl.md:39]:
```suggestion
> + 该操作可以暂停 DDL 作业，但除版本升级外，其他操作和环境变更（例如机器重启、集群重启）不会暂停 DDL 作业。
> + 版本升级时，正在运行的 DDL 作业将被暂停，同时在升级过程中发起的 DDL 作业也将被暂停。升级结束后，所有已暂停的 DDL 作业将恢复执行。升级过程中的操作为自动进行，详情查阅 [TiDB 平滑升级](/smooth-upgrade-tidb.md)。
> + 该操作可以同时暂停多个 DDL 作业，可以通过 [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) 语句来获取 DDL 作业的 `job_id`。
> + 如果希望暂停的作业已经执行完毕或接近执行完毕，暂停操作将失败。
```

File: sql-statements/sql-statement-admin-resume-ddl.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
+---
+title: ADMIN RESUME DDL
+summary: TiDB 数据库中 ADMIN RESUME DDL 的使用概况。
+---
+
+# ADMIN RESUME DDL
+
+`ADMIN RESUME DDL` 语句用于恢复当前处于暂停中的 DDL 作业。可以通过 [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) 语句获取 DDL 作业的 `job_id`。
+
+用于恢复处于暂停中的 DDL 任务。成功恢复后，执行 DDL 任务的 SQL 语句会一直表现为正在执行。恢复一个已经结束的 DDL 任务会在 RESULT 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-resume-ddl.md:10]:
```suggestion
该语句可用于恢复处于暂停中的 DDL 任务。成功恢复后，执行 DDL 任务的 SQL 语句会一直表现为正在执行。如果尝试恢复已经完成的 DDL 任务，会在 `RESULT` 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
```

File: sql-statements/sql-statement-admin-resume-ddl.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+NumList ::=
+    Int64Num ( ',' Int64Num )*
+```
+
+## 示例
+
+可以通过 `ADMIN RESUME DDL JOBS` 语句恢复当前处于暂停中的 DDL 作业，并返回对应作业是否恢复执行：
+
+{{< copyable "sql" >}}
+
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-resume-ddl.md:27]:
```suggestion
```

File: sql-statements/sql-statement-admin-resume-ddl.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+```sql
+ADMIN RESUME DDL JOBS job_id [, job_id] ...;
+```
+
+如果暂停失败，会显示失败的具体原因。
+
+> **注意：**
+>
+> + 该操作可以暂停 DDL 作业，其他所有的操作和环境变更（例如机器重启、集群重启）都不会暂停 DDL 作业。
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-resume-ddl.md:36]: 这条写错了？

File: sql-statements/sql-statement-admin-resume-ddl.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+可以通过 `ADMIN RESUME DDL JOBS` 语句恢复当前处于暂停中的 DDL 作业，并返回对应作业是否恢复执行：
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN RESUME DDL JOBS job_id [, job_id] ...;
+```
+
+如果暂停失败，会显示失败的具体原因。
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-resume-ddl.md:32]:
```suggestion
如果恢复失败，会显示失败的具体原因。
```

File: sql-statements/sql-statement-admin-resume-ddl.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+
+> **注意：**
+>
+> + 该操作可以暂停 DDL 作业，其他所有的操作和环境变更（例如机器重启、集群重启）都不会暂停 DDL 作业。
+> + 版本升级操作发生时，会将正在运行的 DDL 作业暂停；并将升级过程中发起来 DDL 作业也暂停。升级结束后，将所有暂停的 DDL 作业恢复运行
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-resume-ddl.md:37]:
```suggestion
> + 版本升级时，正在运行的 DDL 作业将被暂停，同时在升级过程中发起的 DDL 作业也将被暂停。升级结束后，所有已暂停的 DDL 作业将恢复执行。升级过程中的操作为自动进行，详情查阅 [TiDB 平滑升级](/smooth-upgrade-tidb.md)。
```

File: sql-statements/sql-statement-admin-resume-ddl.md [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+> + 该操作可以同时暂停多个 DDL 作业，可以通过 [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) 语句来获取 DDL 作业的 `job_id`。
+> + 如果希望暂停的作业已经执行完毕或接近执行完毕，暂停操作将失败。
+> + 再次暂停时，会报错 `Error Number: 8260`
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-resume-ddl.md:40]: “暂停”要改成“恢复”？

File: sql-statements/sql-statement-admin-resume-ddl.md [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+
+## MySQL 兼容性
+
+`ADMIN CANCEL DDL` 语句是 TiDB 对 MySQL 语法的扩展。
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-resume-ddl.md:44]:
```suggestion
`ADMIN RESUME DDL` 语句是 TiDB 对 MySQL 语法的扩展。
```

File: TOC.md (hunk 3 above)
Comment (dhysum) [reply TOC.md:688]: ok

File: sql-statements/sql-statement-admin-resume-ddl.md [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+
+如果恢复失败，会显示失败的具体原因。
+
+> **注意：**
+>
+> + 该操作可以恢复已被暂停的 DDL 作业。
+> + 版本升级时，正在运行的 DDL 作业将被暂停，同时在升级过程中发起的 DDL 作业也将被暂停。升级结束后，所有已暂停的 DDL 作业将恢复执行。升级过程中的操作为自动进行，详情查阅 [TiDB 平滑升级](/smooth-upgrade-tidb.md)。
+> + 该操作可以同时恢复多个 DDL 作业，可以通过 [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) 语句来获取 DDL 作业的 `job_id`。
+> + 处于其他状态中的作业无法被恢复，操作将失败。
+> + 再次恢复时时，会报错 `Error Number: 8261`
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-resume-ddl.md:38]:
```suggestion
> + 再次恢复时，会报错 `Error Number: 8261`
```

**Review by ran-huang — APPROVED** [review]

File: sql-statements/sql-statement-admin-resume-ddl.md [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+
+```sql
+ADMIN RESUME DDL JOBS job_id [, job_id] ...;
+```
+
+如果恢复失败，会显示失败的具体原因。
+
+> **注意：**
+>
+> + 该操作可以恢复已被暂停的 DDL 作业。
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-resume-ddl.md:34]:
```suggestion
```

File: sql-statements/sql-statement-admin-resume-ddl.md [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+> + 版本升级时，正在运行的 DDL 作业将被暂停，同时在升级过程中发起的 DDL 作业也将被暂停。升级结束后，所有已暂停的 DDL 作业将恢复执行。升级过程中的操作为自动进行，详情查阅 [TiDB 平滑升级](/smooth-upgrade-tidb.md)。
+> + 该操作可以同时恢复多个 DDL 作业，可以通过 [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) 语句来获取 DDL 作业的 `job_id`。
+> + 处于其他状态中的作业无法被恢复，操作将失败。
+> + 再次恢复时，会报错 `Error Number: 8261`
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-resume-ddl.md:38]:
```suggestion
> + 如果重复恢复同一个 DDL 作业，会报错 `Error Number: 8261`。
```

File: sql-statements/sql-statement-admin-resume-ddl.md [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+如果恢复失败，会显示失败的具体原因。
+
+> **注意：**
+>
+> + 该操作可以恢复已被暂停的 DDL 作业。
+> + 版本升级时，正在运行的 DDL 作业将被暂停，同时在升级过程中发起的 DDL 作业也将被暂停。升级结束后，所有已暂停的 DDL 作业将恢复执行。升级过程中的操作为自动进行，详情查阅 [TiDB 平滑升级](/smooth-upgrade-tidb.md)。
+> + 该操作可以同时恢复多个 DDL 作业，可以通过 [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) 语句来获取 DDL 作业的 `job_id`。
+> + 处于其他状态中的作业无法被恢复，操作将失败。
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-resume-ddl.md:37]:
```suggestion
> + 处于非暂停状态中的作业无法被恢复，操作将失败。
```

File: sql-statements/sql-statement-admin-pause-ddl.md [hunk 18]
```diff
@@ -0,0 +1,47 @@
...
+```
+
+如果暂停失败，会显示失败的具体原因。
+
+> **注意：**
+>
+> + 该操作可以暂停 DDL 作业，但除版本升级外，其他操作和环境变更（例如机器重启、集群重启）不会暂停 DDL 作业。
+> + 版本升级时，正在运行的 DDL 作业将被暂停，同时在升级过程中发起的 DDL 作业也将被暂停。升级结束后，所有已暂停的 DDL 作业将恢复执行。升级过程中的操作为自动进行，详情查阅 [TiDB 平滑升级](/smooth-upgrade-tidb.md)。
+> + 该操作可以同时暂停多个 DDL 作业，可以通过 [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) 语句来获取 DDL 作业的 `job_id`。
+> + 如果希望暂停的作业已经执行完毕或接近执行完毕，暂停操作将失败。
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-pause-ddl.md:37]: L36、L37 这两句话可以挪到 L11

File: sql-statements/sql-statement-admin-resume-ddl.md (hunk 17 above)
Comment (hfxsd) [inline sql-statements/sql-statement-admin-resume-ddl.md:37]: L36、L37 可以挪到 L11

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: error-codes.md [hunk 19]
```diff
@@ -393,6 +393,18 @@ TiDB 兼容 MySQL 的错误码，在大多数情况下，返回和 MySQL 一样
...
+
+    DDL 操作无法被 `admin pause` 暂停运行。
+
+* Error Number: 8261
+
+    DDL 操作无法被 `admin resume` 恢复运行。
+
+* Error Number: 8262
+
+    DDL 已经被 `admin pause` 暂停，无法再次执行。
```
Comment (hfxsd) [inline error-codes.md:406]:
```suggestion
    DDL 操作无法被 `ADMIN PAUSE`` 暂停运行。

* Error Number: 8261

    DDL 操作无法被 `ADMIN RESUME` 恢复运行。

* Error Number: 8262

    DDL 已经被 `ADMIN PAUSE` 暂停，无法再次执行。
```

File: sql-statements/sql-statement-admin.md [hunk 20]
```diff
@@ -212,6 +214,7 @@ ADMIN SHOW DDL JOBS 5 WHERE state != 'synced' AND db_name = 'test';
     * `rollback done`：表示该操作执行失败，回滚完成。
     * `rollingback`：表示该操作执行失败，正在回滚。
     * `cancelling`：表示正在取消该操作。这个状态只有在用 `ADMIN CANCEL DDL JOBS` 命令取消 DDL 作业时才会出现。
+    * `paused`：表示 DDL 已被暂停运行。这个状态只有在用 `ADMIN PAUSED DDL JOBS` 命令暂停 DDL 任务时才会出现。可以通过 `ADMIN RESUME DDL JOBS` 命令进行恢复运行。
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin.md:217]:
```suggestion
    * `paused`：表示 DDL 已被暂停运行。这个状态只有在用 [`ADMIN PAUSED DDL JOBS`](/sql-statements/sql-statement-admin-pause-ddl.md) 命令暂停 DDL 任务时才会出现。可以通过 [`ADMIN RESUME DDL JOBS`](/sql-statements/sql-statement-admin-resume-ddl.md) 命令进行恢复运行。
```

File: sql-statements/sql-statement-admin.md [hunk 21]
```diff
@@ -212,6 +214,7 @@ ADMIN SHOW DDL JOBS 5 WHERE state != 'synced' AND db_name = 'test';
     * `rollback done`：表示该操作执行失败，回滚完成。
     * `rollingback`：表示该操作执行失败，正在回滚。
     * `cancelling`：表示正在取消该操作。这个状态只有在用 `ADMIN CANCEL DDL JOBS` 命令取消 DDL 作业时才会出现。
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin.md:216]:
```suggestion
    * `cancelling`：表示正在取消该操作。这个状态只有在用 [`ADMIN CANCEL DDL JOBS`](/sql-statements/sql-statement-admin-cancel-ddl.md) 命令取消 DDL 作业时才会出现。
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 22]
```diff
@@ -65,6 +65,7 @@ ADMIN SHOW DDL;
     - `rollback done`：表示该操作执行失败，回滚完成。
     - `rollingback`：表示该操作执行失败，正在回滚。
     - `cancelling`：表示正在取消该操作。这个状态只有在用 `ADMIN CANCEL DDL JOBS` 命令取消 DDL 任务时才会出现。
+    - `paused`：表示 DDL 已被暂停运行。这个状态只有在用 `ADMIN PAUSED DDL JOBS` 命令暂停 DDL 任务时才会出现。可以通过 `ADMIN RESUME DDL JOBS` 命令进行恢复运行。
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-show-ddl.md:68]:
```suggestion
    - `cancelling`：表示正在取消该操作。这个状态只有在用 [`ADMIN CANCEL DDL JOBS`](/sql-statements/sql-statement-admin-cancel-ddl.md) 命令取消 DDL 任务时才会出现。
    - `paused`：表示 DDL 已被暂停运行。这个状态只有在用 [`ADMIN PAUSED DDL JOBS`](/sql-statements/sql-statement-admin-pause-ddl.md) 命令暂停 DDL 任务时才会出现。可以通过 [`ADMIN RESUME DDL JOBS`](/sql-statements/sql-statement-admin-resume-ddl.md) 命令进行恢复运行。
```

File: sql-statements/sql-statement-admin-pause-ddl.md [hunk 23]
```diff
@@ -0,0 +1,47 @@
...
+title: ADMIN PAUSE DDL
+summary: TiDB 数据库中 ADMIN PAUSE DDL 的使用概况。
+---
+
+# ADMIN PAUSE DDL
+
+`ADMIN PAUSE DDL` 语句用于暂停当前正在运行的 DDL 作业。可以通过 [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) 语句获取 DDL 作业的 `job_id`。
+
+该语句可用于暂停已经提交但未执行完成的 DDL 任务。成功暂停后，执行 DDL 任务的 SQL 语句不会立即返回，表现为正在执行。如果尝试暂停一个已经完成的 DDL 任务，会在 `RESULT` 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
+
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-pause-ddl.md:11]:
```suggestion

> **警告：**
>
> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。

```

File: sql-statements/sql-statement-admin-resume-ddl.md [hunk 24]
```diff
@@ -0,0 +1,48 @@
...
+title: ADMIN RESUME DDL
+summary: TiDB 数据库中 ADMIN RESUME DDL 的使用概况。
+---
+
+# ADMIN RESUME DDL
+
+`ADMIN RESUME DDL` 语句用于恢复当前处于暂停中的 DDL 作业。可以通过 [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) 语句获取 DDL 作业的 `job_id`。
+
+该语句可用于恢复处于暂停中的 DDL 任务。成功恢复后，执行 DDL 任务的 SQL 语句会一直表现为正在执行。如果尝试恢复已经完成的 DDL 任务，会在 `RESULT` 列看到 `DDL Job:90 not found` 的错误，表示该任务已从 DDL 等待队列中被移除。
+
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-resume-ddl.md:11]:
```suggestion

> **警告：**
>
> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。

```

File: sql-statements/sql-statement-admin-pause-ddl.md [hunk 25]
```diff
@@ -0,0 +1,51 @@
+---
+title: ADMIN PAUSE DDL
+summary: TiDB 数据库中 ADMIN PAUSE DDL 的使用概况。
+---
+
+# ADMIN PAUSE DDL
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-pause-ddl.md:6]: 这里要不要把 JOBS 加上 @dhysum

Comment (dhysum) [reply sql-statements/sql-statement-admin-pause-ddl.md:6]: ok

**Review by benmaoer — APPROVED** [review]
LGTM

File: sql-statements/sql-statement-admin-pause-ddl.md [hunk 26]
```diff
@@ -0,0 +1,51 @@
+---
+title: ADMIN PAUSE DDL
+summary: TiDB 数据库中 ADMIN PAUSE DDL JOBS 的使用概况。
+---
+
+# ADMIN PAUSE DDL
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-pause-ddl.md:6]:
```suggestion
title: ADMIN PAUSE DDL JOBS
summary: TiDB 数据库中 ADMIN PAUSE DDL JOBS 的使用概况。
---

# ADMIN PAUSE DDL JOBS
```

File: sql-statements/sql-statement-admin-resume-ddl.md [hunk 27]
```diff
@@ -0,0 +1,52 @@
+---
+title: ADMIN RESUME DDL
+summary: TiDB 数据库中 ADMIN RESUME DDL 的使用概况。
+---
+
+# ADMIN RESUME DDL
```
Comment (ran-huang) [inline sql-statements/sql-statement-admin-resume-ddl.md:6]:
```suggestion
title: ADMIN RESUME DDL JOBS
summary: TiDB 数据库中 ADMIN RESUME DDL 的使用概况。
---

# ADMIN RESUME DDL JOBS
```

**Pull request merged**